lib/themes_support/menus.php

```diff
@@ -0,0 +1,4 @@
+<?php
+require_once('Classes/DummyThemesSupport.php');
+$config =  array();
+new DummyThemesSupport('menus', $config);
```

Comment:
`$config`is useless, and should be avoided with a default value in the constructor's parameters